app/grandchallenge/algorithms/tasks.py

```diff
@@ -250,7 +252,9 @@ def create_algorithm_jobs(
     return jobs
 
 
-def filter_civs_for_algorithm(*, civ_sets, algorithm_image):
+def filter_civs_for_algorithm(
+    *, civ_sets, algorithm_image, algorithm_model=None
```

Comment:
I think we should be explicit to prevent bugs upstream:

```suggestion
    *, civ_sets, algorithm_image, algorithm_model
```